doc/sphinx/practical-tools/coq-commands.rst

```diff
@@ -226,6 +233,89 @@ and ``coqtop``, unless stated otherwise:
   currently associated color and exit.
 :-h, --help: Print a short usage and exit.
 
+
+
+Compiled interfaces (produced using `-vos`)
+----------------------------------------------
+
+
+Compiled interfaces helps saving time while developing Coq formalizations,
+by compiling the formal statements exported by a library independently of
+the proofs that it contains.
+
+   .. warning::
+
+      Compiled interfaces should only be used for development purposes.
+      At the end of the day, one still needs to proof check all files
+      by producing standard `.vo` files. (Technically, when using `-vos`,
+      fewer universe constraints are collected.)
+
+   .. warning::
+
+      In the current implementation, the use of the command `Include` for
+      importing modules compiled using `-vos` might not work properly.
+
+
+**Typical usage.**
```

Comment:
You can use a sub-section title level (cf. http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections). See also #7510.

Reply:
I'm already in a subsection. I don't like subsubsection so much, it takes too much vertical space i find. Let me know if you think this is really important.

Reply:
It gives some structure. I don't particularly care but @cpitclaudel seemed to care.